### PR TITLE
Update "Transactional Client" Doc

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
$subject

### CODE REVIEW

File: en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md [hunk 1]
```diff
@@ -16,163 +16,208 @@ The Transactional Client EIP controls transactions with the messaging system. It
...
+        [[transport.jms.listener]]
+        name = "myQueueConnectionFactory"
+        parameter.initial_naming_factory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+        parameter.provider_url = "tcp://localhost:61616"
+        parameter.connection_factory_name = "QueueConnectionFactory"
+        parameter.connection_factory_type = "queue"
         ```
 
-4. Copy the following ActiveMQ client JAR files to the `<ESB_HOME>/repository/components/lib` directory. It allows the ESB to connect to the JMS provider.
+4. Download [MessageCounter.jar](https://wso2docs.atlassian.net/wiki/download/attachments/48791778/MessageCounter.jar?version=1&modificationDate=1351260014000&cacheVersion=1&api=v2) and copy it to the `<PROJECT_HOME>/deployment/libs` directory.
```
Comment (chanikag) [inline en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md:125]: Let's add this to github, in mi doc space (under attachments)

File: en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md [hunk 2]
```diff
@@ -16,163 +16,208 @@ The Transactional Client EIP controls transactions with the messaging system. It
...
+                        <m0:price>172.23182849731984</m0:price>
+                        <m0:quantity>18398</m0:quantity>
+                        <m0:symbol>IBM</m0:symbol>
+                    </m0:order>
+                </m0:placeOrder>
+            </soapenv:Body>
+        </soapenv:Envelope>
+        ```
+
+2. Again send the same JMS message again.
```
Comment (chanikag) [inline en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md:190]:
```suggestion
2. Resend the same JMS message. 
```

File: en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md [hunk 3]
```diff
@@ -16,163 +16,208 @@ The Transactional Client EIP controls transactions with the messaging system. It
...
 In many business scenarios, there can be data loss when transferring from one section to another because of crashes and other interruptions. This sample scenario demonstrates how messages containing the MESSAGE_COUNT value (MESSAGE_COUNT=1) are assumed to be causing a failure in mediation, and how transactions guarantee that data will not be lost.
 
-There are two ways to implement transaction in WSO2 ESB as follows.
+There are two ways to implement transactions in WSO2 MI as follows.
 
 - Transaction Mediator
 - JMS Transactions
 
-The ESB can control transactional behavior with a JMS queue and by simulating the Transactional Client EAI pattern. For information on Transactions in WSO2 ESB, refer to Transactions in the WSO2 ESB documentation.
+The WSO2 MI can control transactional behavior with a JMS queue and by simulating the Transactional Client EAI pattern. For information on Transactions in WSO2 MI, refer to [Work with Transactions]({{base_path}}/learn/examples/working-with-transactions/) Documentation.
```
Comment (SanojPunchihewa) [inline en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md:24]:
```suggestion
The WSO2 MI can control transactional behavior with a JMS queue and by simulating the Transactional Client EAI pattern. For more information on transactions in WSO2 MI, refer to the [Work with Transactions]({{base_path}}/learn/examples/working-with-transactions/) documentation.
```

File: en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md [hunk 4]
```diff
@@ -16,163 +16,208 @@ The Transactional Client EIP controls transactions with the messaging system. It
...
+        [[transport.jms.listener]]
+        name = "myQueueConnectionFactory"
+        parameter.initial_naming_factory = "org.apache.activemq.jndi.ActiveMQInitialContextFactory"
+        parameter.provider_url = "tcp://localhost:61616"
+        parameter.connection_factory_name = "QueueConnectionFactory"
+        parameter.connection_factory_type = "queue"
         ```
 
-4. Copy the following ActiveMQ client JAR files to the `<ESB_HOME>/repository/components/lib` directory. It allows the ESB to connect to the JMS provider.
+4. Download [MessageCounter.jar]({{base_path}}/assets/attachments/jar/MessageCounter.jar) and copy it to the `<PROJECT_HOME>/deployment/libs` directory.
```
Comment (SanojPunchihewa) [inline en/docs/learn/enterprise-integration-patterns/messaging-endpoints/transactional-client.md:125]: This step should come after step 15 where we import the Project.

**Review by SanojPunchihewa — APPROVED** [review]

**Pull request merged**